clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/AbstractEnableDisableAtomicOperation.groovy

```diff
@@ -186,8 +188,10 @@ abstract class AbstractEnableDisableAtomicOperation implements AtomicOperation<V
             credentials: credentials,
             region: region,
             asgName: serverGroupName,
-            instanceIds: instanceIds
+            instanceIds: instanceIds,
+            targetHealthyDeployPercentage: description.targetHealthyDeployPercentage ?: 100
```

Comment:
I think (not totally sure) that the groovy truthiness here would convert an explicitly supplied value of `0` to `100`

Reply:
Verified that you're right - going to change this to the more explicit `description.targetHealthyDeployPercentage != null ? description.targetHealthyDeployPercentage : 100`